README-KEAP.md

```diff
@@ -0,0 +1,100 @@
+# Keap DataFlow Fork Modifications
+
+## Testing and Deploying
+
+### Integration
+
+```bash
+gcloud config set project is-events-dataflow-intg
+
+mvn compile exec:java \
+-Dexec.mainClass=com.infusionsoft.dataflow.templates.PubsubToBigQuery \
+-Dexec.cleanupDaemonThreads=false \
+-Dexec.args=" \
+--project=is-events-dataflow-intg \
+--region=us-central1 \
+--workerZone=us-central1-c \
+--stagingLocation=gs://is-events-dataflow-intg/staging \
+--tempLocation=gs://is-events-dataflow-intg/temp \
+--templateLocation=gs://is-events-dataflow-intg/templates/pubsub2BGQ.json \
+--runner=DataflowRunner \
+--defaultWorkerLogLevel=ERROR"
+
+gcloud dataflow jobs run pubsub-to-bigquery \
+--gcs-location=gs://is-events-dataflow-intg/templates/pubsub2BGQ.json \
+--region=us-central1 \
+--zone=us-central1-c \
+--service-account-email=is-events-dataflow-intg@is-events-dataflow-intg.iam.gserviceaccount.com \
+--parameters=inputTopic=projects/is-flagship-events-intg/topics/v1.segment-events-core
+```
+
+Shut down the job after testing [via Cloud Console](https://console.cloud.google.com/dataflow/jobs?authuser=1&project=is-events-dataflow-intg)
+
+-----
+
+### Staging
+
+Update the parameter in `PubsubToBigQuery.java` to point to Stge
+> L157 from `is-events-dataflow-intg` to `is-events-dataflow-stge`
```

Comment:
I would add this to the intg one as well.  Just in case it isn't already seto to `intg`.